### ISSUE TITLE
Fix NaN bug

### ISSUE THREAD

**Issue by cheerfulstoic** [cheerfulstoic/kommun_compare#1]
When unchecking everything but "El och fjärrvärme" (as an example), some `NaN` values appear in the table

**Comment by Greenheart** [comment]
Here's an image of the bug

![image](https://user-images.githubusercontent.com/6125097/74988102-39579200-543d-11ea-97ea-a3432516e4f8.png)

**Comment by Greenheart** [comment]
Is this resolved?

**Comment by cheerfulstoic** [comment]
It is... not...  Though I have an as-yet unanswered question about what we should show because I'm not sure how to calculate what the percentage is when you go from something to nothing (like 1% to 0%).  I'm not even sure we're calculating these last two columns correctly...

**Comment by Greenheart** [comment]
Ah, I see. I'm don't know. Perhaps hide the number or just show "-"?

**Comment by cheerfulstoic** [comment]
Yeah, that might be good.  Jan just made some changes to the formula which I haven't looked at too closely, though I'm seeing more NaNs now with those changes, so we should figure those out.  I would @ him if I knew what his username was 😄

**Comment by Greenheart** [comment]
@janlindblad
